### ISSUE TITLE
Flank v20.09.0 404 not found

### ISSUE THREAD

**Issue by Voto** [Flank/flank#1075]
Hello there, our Bitrise CICD is currently running into trouble downloading the newest update of flank.jar.

```
+------------------------------------------------------------------------------+
| (10) flank                                                                   |
+------------------------------------------------------------------------------+
| id: flank                                                                    |
| version: 0.1.2                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2020-09-01T14:52:41Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
Config:
- ServiceAccountJSON: *****
- ConfigPath: ./edeka2020/flank.yml
- Version: latest
- CommandFlags: 
Downloading binary
Failed to download binary, error: http GET https://github.com/Flank/flank/releases/download/v20.09.0/flank.jar non success status code: 404
```

We saw this yesterday already and thought that wait and drink tea might solve the issue, because of the new release but seems to be still an issue today.

Thanks 
Alex

**Comment by adamfilipow92** [comment]
Hello, thanks for your feedback. Our release action is broken and we working on a fix.

**Comment by Voto** [comment]
Which Flank version is stable for usage? Since #1073 20.08.4 seems to have problems too

**Comment by adamfilipow92** [comment]
Hello, snapshots and release processes working now.
https://github.com/Flank/flank/releases
We have released v20.09.0
Thanks for your patience!
